### PR TITLE
Blog post options taxi

### PR DESCRIPTION
**Pull request by TheTanc**

### CODE REVIEW

File: blog/2020-09-24-peformance-matters.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
```
Comment (mpsq) [inline blog/2020-09-24-peformance-matters.md:1]: This file should not be new, did you rebase against master?

File: blog/2020-10-16-taxi-drivers-are-options-traders.md [hunk 2]
```diff
@@ -0,0 +1,287 @@
+---
+title: NYC cab drivers, the unsuspecting options traders
+author: Tancrede Collard
+author_title: QuestDB Team
+author_url: https://github.com/TheTanc
+author_image_url: https://avatars.githubusercontent.com/TheTanc
+tags: [nyc taxi dataset, options]
```
Comment (mpsq) [inline blog/2020-10-16-taxi-drivers-are-options-traders.md:7]:
```suggestion
tags: [deep-dive, story]
```

File: blog/2020-10-16-taxi-drivers-are-options-traders.md [hunk 3]
```diff
@@ -0,0 +1,287 @@
...
+description: An experiment analyzing the NYC taxi dataset through the eye of an options trader.
```
Comment (mpsq) [inline blog/2020-10-16-taxi-drivers-are-options-traders.md:8]:
```suggestion
description: An experiment analyzing the NYC taxi dataset through the eyes of an options trader.
```

File: blog/2020-10-16-taxi-drivers-are-options-traders.md [hunk 4]
```diff
@@ -0,0 +1,287 @@
...
+
+Every cab I have ever ridden complained about how hard it is for drivers to make ends meet. The public is generally quick to 
+blame unfair competition from the likes of Uber. But other forces share the responsibility for this situation. 
+
+In this post, I focus on the impact the antiquated meter system had on the livelihood of the beloved NYC cabbies by drawing an analogy with options trading.
+Interestingly, this approach allows us to show that drivers are worse-off, independently of competition.
+
+<!--truncate-->
+
+A couple of weeks ago, I was preparing data for the demo we are hosting of QuestDB which you can try [here](http://try.questdb.io:9000/).
```
Comment (mpsq) [inline blog/2020-10-16-taxi-drivers-are-options-traders.md:19]: Could you rename the [here] link? This is not optimal for good user experience.

File: blog/2020-10-16-taxi-drivers-are-options-traders.md [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+## Modeling variable earnings
+
+Let's assume a cab is driving at a constant speed for an hour. At the end of the hour, the driver can expect to earn 
+- $30 if they drove below 12mph ($0.50 a minute)
+- $2.50 x their average speed if they drove above 12mph
+
+Let's plot the hourly earnings in function of speed. This instantly reminds me of an old friend: call options!
+
+<img
+  alt="A chart of call option payoff showing how cab drivers earnings increase with their average realised driving speed"
```
Comment (mpsq) [inline blog/2020-10-16-taxi-drivers-are-options-traders.md:46]:
```suggestion
  alt="A chart of call option payoff showing how cab drivers earnings increase with their average realized driving speed"
```

File: blog/2020-10-16-taxi-drivers-are-options-traders.md [hunk 6]
```diff
@@ -0,0 +1,287 @@
...
+```Hourly Fare = 30 + max(0, Speed - 12)```
+
+Interestingly, the above notation breaks down the hourly variable fare into two components.
+- A guaranteed component `30`: whenever driving a customer, a cab will make a guaranteed $30 an hour. 
+- An optional component `max(0, Speed - 12)`: The driver can make extra money by driving the customer around faster. 
+
+Graphically, the breakdown between `guaranteed` and `variable` fare components look like the below:
+
+<img
+  alt="A chart of call option payoff showing how cab drivers earnings increase with their average realised driving speed broken down between fixed and variable"
```
Comment (mpsq) [inline blog/2020-10-16-taxi-drivers-are-options-traders.md:62]:
```suggestion
  alt="A chart of call option payoff showing how cab drivers earnings increase with their average realized driving speed broken down between fixed and variable"
```

File: blog/2020-10-16-taxi-drivers-are-options-traders.md [hunk 7]
```diff
@@ -0,0 +1,287 @@
...
+But in a world with increased supply (Uber, Lyft etc.), I think it is safe to assume that the wait time for drivers has increased. 
+I don’t know for sure whether the wait time is above 5 minutes, but my guess is that it is. So the effect as a deterrent is somewhat nullified.
+
+If drivers are uncertain about their likelihood of finding the next ride, 
+and if the optional component of the fare has become an insignificant fraction of their earnings (most of which based on time being occupied), 
+then it makes more sense to drive slow, and to hold on to the current customer for as long as possible. In the end, $30 an hour is better than 0.
+
+## Your turn to explore the data
+
+We made this dataset and the database available online. You can query it directly from your browser [here](http://try.questdb.io:9000/).
```
Comment (mpsq) [inline blog/2020-10-16-taxi-drivers-are-options-traders.md:277]: Same as above, [here] links are not recommanded

File: blog/2020-10-16-taxi-drivers-are-options-traders.md [hunk 8]
```diff
@@ -0,0 +1,287 @@
...
+
+The vast majority of rides are priced using the [standard meter system](https://www1.nyc.gov/site/tlc/passengers/taxi-fare.page). 
+A meter calculates a fare based on time, speed, and distance. Then, depending on your route and when you are riding, it adds a number of taxes, tolls and surcharges.
+
+Most of the driver's earnings come from the meter: a flat $2.50 for entering the cab, and a variable fare which is a function of speed, time and distance calculated as follows:
+- $2.50 per mile when driving above 12mph
+- $0.50 per minute otherwise (so the driver earns a living in case a customer takes them through traffic).
+
+This post focuses on the variable fare (the output of the meter excluding the $2.50 start fee). 
+We normalise it as an `hourly rate` assuming drivers are constantly driving a customer.
```
Comment (mpsq) [inline blog/2020-10-16-taxi-drivers-are-options-traders.md:34]:
```suggestion
We normalize it as an `hourly rate` assuming drivers are constantly driving a customer. 
```

File: blog/2020-10-16-taxi-drivers-are-options-traders.md [hunk 9]
```diff
@@ -0,0 +1,287 @@
...
+|4|16.66%|2|0.3332|
+|5|16.66%|3|0.4998|
+|6|16.66%|4|0.6664|
+
+By adding all the potential payouts weighed by their probability, we compute the expected value of playing this game $1.4494. 
+- If we pay less to play the game, we will make money over time. 
+- If we pay more, we lose in the long run. 
+
+This example shows that in its simplest form, the value of an option is equal to the product of the payout profile and the associated probability distribution. 
+Let’s visualise this by plotting  the values for our game in the following chart:
```
Comment (mpsq) [inline blog/2020-10-16-taxi-drivers-are-options-traders.md:102]:
```suggestion
Let’s visualize this by plotting  the values for our game in the following chart:
```

**Review by mpsq — CHANGES_REQUESTED** [review]

File: blog/2020-10-16-taxi-drivers-are-options-traders.md [hunk 10]
```diff
@@ -0,0 +1,287 @@
...
+  className="screenshot--shadow screenshot--docs"
+  src="/img/blog/2020-10-16/variable-hourly-fare-over-time.png"
+/>
+
+
+I don’t have the data to tell why the average speed is lower, but I would think this could be attributed to more vehicles on the road as a result of Uber, Lyft, and other FHV, along with urban planning, for example cycle lanes making for less space on the road and more congestion.
+
+The impact is significant. Over the past 10 years, increased traffic has cost up to $10 an hour per taxi. To put this in context, this means $29,000 per driver per year (8 hours a day, no holidays), or 300 million dollars a year for the NYC cab industry! And these are lower bound numbers. In reality, drivers share cabs. If we assume all of the 13,500 cabs are constantly on the road, this adds up to 1.2 billion dollars a year lost for the industry!
+
+## Customers are losing too.
```
Comment (mpsq) [inline blog/2020-10-16-taxi-drivers-are-options-traders.md:246]:
```suggestion
## Customers are losing too
```

File: blog/2020-09-24-peformance-matters.md (hunk 1 above)
Comment (TheTanc) [reply blog/2020-09-24-peformance-matters.md]: I did but nevermind. Deleted the file

**Review by mpsq — APPROVED** [review]

**Pull request merged**